kotlinx-coroutines-debug/src/CoroutineInfo.kt

```diff
@@ -94,6 +90,10 @@ public data class CoroutineInfo internal constructor(
             lastObservedThread = null
         }
     }
+
+    override fun toString(): String {
+        return "CoroutineInfo(state=$state,context=$context)"
+    }
```

Comment:
It would read in a much more Kotlin way (less boilerplate to parse to eyes) like this:
```
    override fun toString(): String = "CoroutineInfo(state=$state,context=$context)"
```